### DISCUSSION TITLE
Switch between Github.com accounts just like bitbucket

### DISCUSSION THREAD

**Discussion by aneesdev** [community/community#29945] [Other Feature Feedback, Questions, & Ideas]
Hi,

I really like github.com for my code. I have multiple accounts on github.com if I need to switch to another account on github.com. i need to log out and then log in the new one. or I need to do on incognito browser.

I think github.com should add the switch accounts just like bitbucket like this:

![image](https://user-images.githubusercontent.com/92947803/185419292-85badae4-fbc3-4a4e-a654-7ce455c053c4.png)

**Comment by NikhilMath** [comment]
I don't think there is anything like that to help. 
You can always add another email to your current account (which I think GitHub is pushing their customers to do).

**Reply by NikhilMath** [reply]
and that is currently what I do.

**Reply by NikhilMath** [reply]
you can have different repo updates send updates to different emails.

**Reply by reinhart1010** [reply]
There might be some valid use cases on why is this necessary, like [Signal](https://signal.org) staff commonly have a separate account (see https://github.com/signalapp/Signal-Android/commits/main) on public GitHub.com (not GitHub Enterprise). I also maintain some bot accounts who do automated commits, which unfortunately, (depends on how you see it) pollutes my contribution graph everyday.

![image](https://user-images.githubusercontent.com/17312341/186112827-84f05688-c006-4387-9035-80c0edff9bed.png)

It's a little bit unfair when I can be active on GitHub without having to be active on GitHub, so I decided to create another GitHub account (@1010bots) for automated commits and so.

![image](https://user-images.githubusercontent.com/17312341/186113205-9bebbd0c-f91e-48b1-adc7-6667e3d3e9a9.png)

**Reply by NikhilMath** [reply]
yeah that makes sense. yeah I can see how having a bot messes up your personal commit graph. It because not trustworthy anymore. That is something I never thought about. I like how you shared that with me.
